plugins/catalog/src/components/EntityOrphanWarning/EntityOrphanWarning.tsx

```diff
@@ -44,9 +41,9 @@ export const EntityOrphanWarning = () => {
     <>
       <Alert severity="warning" onClick={() => setConfirmationDialogOpen(true)}>
         This entity is not referenced by any location and is therefore not
-        receiving updates. Click here to unregister.
+        receiving updates. Click here to delete.
```

Comment:
This is pretty strange UX since the alert doesn't look clickable and there's no link/button styling. I'd suggest having a `Delete` button within the alert instead of the `Click here to delete` text.

Reply:
Yeah we talked about it actually and I said the same thing that a right aligned button would be nicer eventually but then that needs a bit of flexboxin' and stuff so I suggested postponing and calling it a day